shark_turbine/kernel/wave/expansion.py

```diff
@@ -19,7 +19,11 @@
 from .._support.indexing import IndexingContext, IndexSequence
 from ...support.logging import get_logger
 from .._support.tracing import CapturedTrace
-from .utils import get_mma_dimensional_mapping, specialize_index_sequence
+from .utils import (
+    get_mma_dimensional_mapping,
+    specialize_index_sequence,
+    remove_chained_getresult,
+)
 from ..lang.global_symbols import *
 
 logger = get_logger("turbine.wave.expansion")
```

Comment:
Could you update the comment below now that the type of expandednodemap has changed?

Reply:
done!